components/grid-col/src/stories.js

```diff
@@ -110,3 +110,17 @@ examples.add('Quarters', () => (
     </GridRow>
   </Fragment>
 ));
+
+examples.add('One Quarter and autoFill', () => (
+  <Fragment>
```

Comment:
This Fragment isn’t needed as it contains only a single GridRow

Reply:
Good point, it's not needed for any of the others too, removing.